src/menu.stories.ts

```diff
@@ -262,9 +275,10 @@ const meta: Meta = {
         ? nothing
         : arguments_.placement}
       size=${arguments_.size === 'large' ? nothing : arguments_.size}
+      ?loading=${arguments_.loading}
       ?open=${arguments_.open}
     >
-      <glide-core-button label="Target" slot="target"></glide-core-button>
+      <glide-core-button label="Toggle" slot="target"></glide-core-button>
```

Comment:
Renamed it to "Toggle" so it's in line with Modal's "Open" button and Drawer's "Toggle" button. 

Menu is a bit different from those two in that its button does into its "target" slot. But I thought "Toggle" would still make more sense from the Storybook user's perspective. Let me know what you think.